content/en/real_user_monitoring/android/mobile_vitals.md

```diff
@@ -17,13 +17,15 @@ Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android
 
 You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
 
-{{< img src="real_user_monitoring/android/android_mobile_vitals-1.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+{{< img src="real_user_monitoring/android/android_mobile_vitals-2.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
 
 To access the RUM mobile app performance dashboard, scroll down to the Monitor Performance Metrics section from the application summary page and click **Performance Overview**.
 
 {{< img src="real_user_monitoring/android/mobile-performance-dashboard-3.png" alt="Mobile Vitals in the Performance Tab" style="width:90%;">}}
 
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
+
+{{< img src="real_user_monitoring/android/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```

Comment:
can we take the screenshot when the popover is open after clicking on a metric?

Reply:
@timmld Updated the screenshot - is this what you meant?

Reply:
exactly, thanks!